scripts/travis/openj9-test.sh

```diff
@@ -5,13 +5,13 @@ cd ../..
 failures=""
 
 docker build -t criu-openj9-ubuntu-test:latest -f scripts/build/Dockerfile.openj9-ubuntu .
-docker run --rm --privileged criu-openj9-ubuntu-test:latest
+docker run -p 49200:49200 --rm --privileged criu-openj9-ubuntu-test:latest
 if [ $? -ne 0 ]; then
 	failures=`echo "$failures ubuntu"`
 fi
 
 docker build -t criu-openj9-alpine-test:latest -f scripts/build/Dockerfile.openj9-alpine .
-docker run --rm --privileged criu-openj9-alpine-test:latest
+docker run -p 49200:49200 --rm --privileged criu-openj9-alpine-test:latest
```

Comment:
What is the reason to export the port? For local testing? It does not seem to be used here.

Reply:
I have removed it.